Document running it tests from intellij IDE (#12440)
apache/druid: integration-tests/README.md

   mvn verify -P integration-tests -pl integration-tests -Ddocker.build.skip=true -Ddocker.run.skip=true -Dit.test=<test_name>
   ```

### Running tests from IntelliJ

> A prerequisite to run tests from intellij you need a cluster up and running. Modify the run configuration's `VM options`
> to the values below.

Reviewer: ```suggestion
Before running tests from IntelliJ, ensure you have a Druid cluster running.
In IntelliJ, edit configurations to add the following VM options:
```

Author: Reads a lot better. Will just make slight naming changes to make sure it matches exactly with intellij's naming

```
-Duser.timezone=UTC
-Dfile.encoding=UTF-8
-Ddruid.test.config.dockerIp=localhost
-Ddruid.zk.service.host=localhost
-Ddruid.client.https.trustStorePath=client_tls/truststore.jks
-Ddruid.client.https.trustStorePassword=druid123
-Ddruid.client.https.keyStorePath=client_tls/client.jks
-Ddruid.client.https.certAlias=druid
-Ddruid.client.https.keyManagerPassword=druid123
-Ddruid.client.https.keyStorePassword=druid123
```

> and to run the test you need to run the test from the test configuration often times found around the top right corner

Reviewer: ```suggestion
Run tests from the test configuration found in the top right corner of the IDE.
```
Is there a certain icon or menu item to look for?
